int32 id
int32 class_id
string class_name
geometry_msgs/Point center
geometry_msgs/Point min_bbox
geometry_msgs/Point max_bbox
BBox2d bbox2d